src/java/picard/analysis/CollectJumpingLibraryMetrics.java

```diff
@@ -55,13 +55,46 @@
-        usage = "Computes jumping library metrics.  Gets all data for computation from the first" +
-                "read in each pair and assumes that the MQ tag is set with the mate's mapping quality.  If the " +
-                "MQ tag is not set, then the program assumes that the mate's mapping quality is >= MINIMUM_MAPPING_QUALITY",
-        usageShort = "Produces jumping library metrics for the provided SAM/BAMs",
+        usage = CollectJumpingLibraryMetrics.USAGE_SUMMARY + CollectJumpingLibraryMetrics.USAGE_DETAILS,
+        usageShort = CollectJumpingLibraryMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectJumpingLibraryMetrics extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "SAM file generated with a jumping library, produced by the " +
+            "CollectJumpingLibraryMetrics program.";
+    static final String USAGE_DETAILS = "The CollectJumpingLibraryMetrics tool collects high-level metrics about the " +
+            "presence of outward- (jumping) and inward-facing (non-jumping) read pairs within a SAM/BAM file.<br /><br />" +
+            "  Jumping libraries are created to bypass difficult to align/map regions, such as those containing repetitive" +
+            " DNA sequences.  Briefly, the DNA of interest is identified, cut into fragments either with restriction" +
+            " enzymes or by shearing, size-selected, and ligated to adapters for bead-capture and circularized.  " +
+            "After bead-capture, the DNA linearized via restriction enzymes and can be sequenced using adapter" +
+            " primers facing in outward [reverse/forward (RF)] directions.  " +
+            "These library inserts are considered jumping because the ends originate from distal genomic DNA sequences" +
+            " and are ligated adjacent to one another during circularization.  Potential artifacts of this method " +
+            "include small inserts (lacking the linearizing restriction enzyme sequence), which are inward" +
+            " facing [forward/reverse (FR)] (non-jumping) read pairs.  In addition, chimeras result from the" +
+            " paired ends falling on different chromosomes, the insert size exceeding the maximum of 100 KB, or two" +
+            " times the mode of the insert size for outward-facing pairs. Please see: " +
+            "<li> www.wikipedia.org/wiki/Jumping_library#Paired-end_sequencing for additional information." +
+                    "" +
+                    "This program gets all data for computation from the first read in each pair and assumes that the mapping " +
+            "quality (MQ) tag is set with the mate's mapping quality.  If the MQ tag is not set, then the program " +
+            "assumes that the mate's MQ is greater than or equal to MINIMUM_MAPPING_QUALITY (default value is 0).<br /><br />  " +
+            "Tool produces (txt) files with the extension \".jump_metrics\".  Add the (.txt) suffix to open file" +

```

Comment:
Also, extensions should be specified uniformly, so if you use \".jump_metrics\", should use \".txt\" instead of (.txt)